nautobot/dcim/forms.py

```diff
@@ -473,7 +473,18 @@ class LocationForm(NautobotModelForm, TenancyForm):
 
     class Meta:
         model = Location
-        fields = ["location_type", "parent", "site", "name", "slug", "status", "tenant_group", "tenant", "description"]
+        fields = [
+            "location_type",
+            "parent",
+            "site",
+            "name",
+            "slug",
+            "status",
+            "tenant_group",
+            "tenant",
+            "description",
+            "tags",
```

Comment:
`tag` on the FilterForm, `tags` on the Form. Is this intended?

Reply:
```suggestion
            "tag",
```

Reply:
split brain between updates I made vs Glenn, I will update

Reply:
Yeah, this was correct before - "tags" is the model field, so it's the correct name on the Form.

For the FilterForm, see my comment:

> Location tag filter name of `tags` was correct with how we want to do things moving forward, but there's existing code (notably the implementation of TagColumn) that expects this filter to be named `tag` as it is (incorrectly) named in all other existing filters. So sadly I changed it to `tag`.

Reply:
my bad :)

Reply:
More like _my_ bad. I should have read @glennmatthews's update of the PR description.